suite-common/toast-notifications/src/types.ts

```diff
@@ -158,9 +164,15 @@ export interface CommonNotificationPayload {
     error?: string;
 }
 
-export type ToastNotification = { context: 'toast' } & CommonNotificationPayload & ToastPayload;
-export type EventNotification = { context: 'event' } & CommonNotificationPayload &
-    NotificationEventPayload;
+type ToastContext = { context: 'toast' };
+type EventContext = { context: 'event' };
+
+export type ToastNotification = ToastContext & CommonNotificationPayload & ToastPayload;
```

Comment:
nit: kind of makes sense to now rename it to `DesktopToastNotification` maybe?

Reply:
I would prefer to not touch anything that is used in the desktop version if you don't insist on this.